jets/c/ripn.c

```diff
@@ -0,0 +1,167 @@
+/* j/c/ripn.c
+**
+*/
+#include "all.h"
+
+#define MIN(x,y) ((x)>(y) ? (y) : (x))
+
+#define DROPBITS(n,w)      ((w) >> (n))
+#define TAKEBITS(n,w)      ((w) & ((1 << (n)) - 1))
+#define BITSLICE(n,off,w)  TAKEBITS(n, DROPBITS(32-(off+n), (w)))
+
+#define MERGE(n,ins,sig)  ((ins) | (sig) << n)
+
+/*
+  TAKEBITS(3,0xff) -> 0x5
```

Comment:
This is really a nitpick, but could you reorder these two examples to be in the same order as the definitions? I just spent a few confused minutes thinking TAKEBITS was DROPBITS, and I probably won't be the last.